website/index.ejs

```diff
@@ -518,6 +518,12 @@
           <p>Various blocks for interacting with the network. Created by <a href="https://scratch.mit.edu/users/TrueFantom/">TrueFantom</a>.</p>
         </div>
 
+        <div class="extension">
+          <%- banner('CubesterYT/ScratchHook') %>
+          <h3>ScratchHook</h3>
+          <p>A fully modular extension that allows you to use Webhooks, with documentation <a href="https://github.com/CubesterYT/dependencies/blob/main/TurboWarp%20Extensions/ScratchHook/ScratchHook.md">here</a>. Created by <a href="https://scratch.mit.edu/users/CubesterYT/">CubesterYT</a>.</p>
```

Comment:
if you have documentation then set a docsURI https://docs.turbowarp.org/development/extensions/assorted-apis#docsuri

Reply:
usually we want these to be hosted on extensions.turbowarp.org though, so you could put an HTML file in website/CubesterYT/ScratchHook.html and then use https://extensions.turbowarp.org/CubesterYT/ScratcHook.html

Reply:
Is it alright for me to have it hosted on my own for now? I'll add the html documentation version soon.

Reply:
HTML Docs added